index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>My portoflio</title>
+</head>
+<body>
+    <nav class="nav_bar">
+        <div class="logo"> <h2>Mylogo</h2></div> 
+            </div class="nav_list">
+                <ul>
+                    <li><a href="#">portflio</a></li>
+                    <li><a href="#">about</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+                <img class="burger-icon" src="img/Union.png" alt="bruger-icon">
+
+        </div>
+    </nav>
+    <div class="section">
+        <div class="text">
+                <h1 >I’m Alexandra.<br>Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                    <p id="lets-con">LET’S CONNECT</p>
+                        <div class="img-icons">
+                            <img src="img/twitter.jpg" alt="twitter icon">
+                            <img src="img/Linkedin icon.jpg" alt="Linked in icon">
+                            <img src="img/medium.jpg" alt="medium icon">
+                            <img src="img/github.jpg" alt="github icon">
+                            <img src="img/angellist.jpg" alt="angellist icon">
+                        </div>
+        </div>  
+    </div>
+    </section>
```

Comment:
- [ ] Excellent job designing the HTML layout for your headline section 🚀. However, please kindly make sure that you use the `main` semantic tag to wrap all your sections and `section` tags to wrap all your sections, including the headline, currently, you are using a `div` tag as the main wrapper for your headline but it will be a good idea to use a `section` tag for this instead and then include that section inside your `main` tag 🤓.
This is a small example of a semantic HTML design 👇🏽, you can read more information about it [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

```
<html>
    <head>
        <title>Portfolio</title>
    </head>
    <body>
        <header>
            Here goes logo, navigation, etc.
        </header>
        <main>
            <section>
                Here you can include your headline
            </section>
            <section>
                Here you can include My recent works
            </section>
        </main>
        <footer>
            Footer information, links, etc.
        </footer>
    </body>
</html>
```